src/pocketmine/block/SeaLantern.php

```diff
@@ -49,4 +49,4 @@ public function getDrops(Item $item){
 		];
 	}
 
-}
```

Comment:
@robske110 why? It fixes a bug. 

Sea Lanterns Transparent:
![2017-02-01 7](https://cloud.githubusercontent.com/assets/4651084/22539441/54fe9064-e8d7-11e6-87c2-99b226640790.png)
Sea Lanterns Solid:
![2017-02-01 8](https://cloud.githubusercontent.com/assets/4651084/22539442/5502e2c2-e8d7-11e6-8e08-c0a6f685dd31.png)
Sea Lanterns Transparent:
![2017-02-01 6](https://cloud.githubusercontent.com/assets/4651084/22539451/6104a3b2-e8d7-11e6-8265-d20c18772897.png)
Sea Lanterns Solid:
![2017-02-01 5](https://cloud.githubusercontent.com/assets/4651084/22539452/6104caea-e8d7-11e6-8399-b0ea49ed3b6d.png)

The top half of the Rainbow/DSX building is dark because Sea Lanterns are Solid with lightFilter 1 instead of 15

`				      	if($block->isSolid()){

						if($block->isTransparent()){

							if($block instanceof Liquid or $block instanceof Ice){

								self::$lightFilter[$id] = 2;

							}else{

								self::$lightFilter[$id] = 1;

							}

						}else{

							self::$lightFilter[$id] = 15;

						}

					}else{

						self::$lightFilter[$id] = 1;

					}`